iroh-embed/src/lib.rs

```diff
@@ -63,6 +96,12 @@ pub struct IrohBuilder {
     indexer: Option<IndexerUrl>,
 }
 
+impl Default for IrohBuilder {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 impl IrohBuilder {
     /// Creates a new [`IrohBuilder`].
     pub fn new() -> Self {
```

Comment:
This can be removed in favour of `IrohBuilder::default()`!

Reply:
Is it good style not to provide new in this case?  I'm used to always providing both constructors, clippy's https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default kind of also suggests it is the usual style.

Reply:
having `new` just call `default` is pretty nice as an api imo

Reply:
Ah! I always use only either `new` with arguments or `default` if there are none. I messed up the way clippy suggests in my head...

Feel free to ignore my comment! :laughing: